crates/duckdb/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "duckdb"
-version = "0.10.2"
+version = "1.0.0"
```

If `duckdb` and `libduckdb-sys` share the same version, this can simply be defined in the root manifest.

```suggestion
version.workspace = true
```